src/app/public/modules/timepicker/timepicker.directive.ts

```diff
@@ -66,9 +66,19 @@ export class SkyTimepickerInputDirective implements
   public pickerChangedSubscription: Subscription;
   private _timeFormat: string = 'hh';
 
+  /**
+   * Creates the timepicker input field and picker. Place this attribute on an `input` element,
+   * and wrap the input in a `sky-timepicker` component.
+   * This attribute must be set to the instance of the `sky-timepicker`.
```

Comment:
Is the last sentence here correct? The last sentence of the corresponding directive description for datepicker says `The value that users select is driven through the ngModel attribute specified on the input element.`, and we have that same sentence at the top of the existing Development tab for timepicker. Should that sentence be here as well? And if the current last sentence is correct, should it also be in place on the corresponding description in the datepicker directive docs? It seems like these two directives should work more or less the same way.

Reply:
You're right to point that out, but the answer, sadly, is no... they are different in implementation. At some point we updated datepicker but never made the same changes to timepicker.